prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -128,9 +128,14 @@ public async Task<IEnumerable<EnrolleeListViewModel>> GetEnrolleesAsync(Enrollee
                     .SearchCollections(e => e.Certifications.Select(c => c.LicenseNumber))
                     .Containing(searchOptions.TextSearch)
                 )
-                .If(searchOptions.StatusCode.HasValue, q => q
+                .If(searchOptions.StatusCode.HasValue && searchOptions.StatusCode != 42, q => q
```

Comment:
42 is not seen in the seeded table configuration `StatusConfiguration` or data migration, is it just temporary?

Reply:
They wanted this item in the dropdown and they wanted it for 1 point. Just the way to get it in. Chose 42 since i doubt we will ever have that many status.

Reply:
We don't want this is the status table since it is not a status